__tests__/app.test.js

```diff
@@ -56,4 +57,33 @@ describe("API", () => {
         });
     });
   });
+
+  //GET /API/ARTICLES
+  describe("GET/api/articles - Display Array of article objects", () => {
+    it("200: GET responds with an array of article objects, each of which to have appropriate properties", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          console.log(body);
+          expect(body.articles).toBeInstanceOf(Array);
+          expect(body.articles).toHaveLength(12);
+          expect(body.articles).toBeSortedBy("created_at", {
+            descending: true,
+            coerce: false,
+          });
+          body.articles.forEach((article) => {
```

Comment:
While the hasOwnProperty works here, for future tickets it might be worth looking into the objectContaining matcher in Jest so we can check both the properties and the data types of the values at the same time.